### PR TITLE
Added Worldcat module and util for issue #152

### PR DESCRIPTION
**Pull request by vikas-kundu**
Here is a gif of the same: https://asciinema.org/a/01gpKnkcCM4TydeDVnzn1VRkx

### CODE REVIEW

File: modules/search/worldcat.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from xml.etree import ElementTree
```
Comment (saeeddhqan) [inline modules/search/worldcat.py:18]: Add it to the requirement file

**Review by saeeddhqan — COMMENTED** [review]
looks good. check the amendments and let me know

File: modules/search/worldcat.py (hunk 1 above)
Comment (vikas-kundu) [reply modules/search/worldcat.py:18]: It is present in the basic python library. Do i still need to add it to the requirements ?

**Comment by vikas-kundu** [comment]
> looks good. check the amendments and let me know

Fixed minor bugs, works fine now.

File: modules/search/worldcat.py (hunk 1 above)
Comment (vikas-kundu) [reply modules/search/worldcat.py:18]: Here, it is a part of the standard python library : https://docs.python.org/3/library/xml.etree.elementtree.html

Comment (saeeddhqan) [reply modules/search/worldcat.py:18]: well, I check this page: https://docs.python.org/3/py-modindex.html and found it isn't in the list

Comment (vikas-kundu) [reply modules/search/worldcat.py:18]: It is present on this page under the xml tab if you will expand it. Here is a screenshot https://imgur.com/a/xcxMxE1

Comment (saeeddhqan) [reply modules/search/worldcat.py:18]: got it

**Pull request merged**